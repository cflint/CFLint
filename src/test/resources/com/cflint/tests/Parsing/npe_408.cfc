<cfscript>
	// Loop over each CSS pair.
	for (
		local.PairIterator = local.Pairs.Iterator() ;
		local.PairIterator.HasNext() ;
		){
	}
</cfscript>