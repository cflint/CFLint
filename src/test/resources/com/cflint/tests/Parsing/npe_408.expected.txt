{
  "version" : "",
  "timestamp" : 0,
  "issues" : [ {
    "severity" : "ERROR",
    "id" : "PARSE_ERROR",
    "message" : "PARSE_ERROR",
    "category" : "CFLINT",
    "abbrev" : "PE",
    "locations" : [ {
      "file" : "src/test/resources/com/cflint/tests/Parsing/npe_408.cfc",
      "fileName" : "npe_408.cfc",
      "function" : "",
      "column" : 0,
      "line" : 1,
      "message" : "Unable to parse",
      "variable" : "",
      "expression" : "<cfscript>/r/n/t/ Loop over each CSS pair./r/n/tfor (/r/n/t/tlocal.PairIterator = local.Pairs.Iterator() ;/r/n/t/tlocal.PairIterator.HasNext() ;/r/n/t/t){/r/n/t}/r/n</cfscript>"
    } ]
  } ],
  "counts" : {
    "totalFiles" : 0,
    "totalLines" : 0,
    "countByCode" : [ {
      "code" : "PARSE_ERROR",
      "count" : 1
    } ],
    "countBySeverity" : [ {
      "severity" : "ERROR",
      "count" : 1
    } ]
  }
}